redbot/core/core_commands.py

```diff
@@ -1229,70 +1231,67 @@ async def perms(self, ctx, level: int):
         await ctx.send("The new permissions level has been set.")
 
     @commands.command()
-    @commands.guild_only()
     @checks.is_owner()
-    async def leave(self, ctx: commands.Context):
-        """Leaves the current server."""
-        await ctx.send(_("Are you sure you want me to leave this server? (y/n)"))
+    async def leave(self, ctx: commands.Context, *server_ids: GuildConverter):
+        """Leaves a server.
```

Comment:
One of these would work better here:
```suggestion
        """Leaves servers.
```
```suggestion
        """Leaves the passed servers.
```